configs/rwkv/430M.yml

```diff
@@ -0,0 +1,103 @@
+{
+  # Parallelism is not yet supported for rwkv
+  "pipe_parallel_size": 1,
+  "model_parallel_size": 1,
+
+  "num_layers": 24,
+  "hidden_size": 1024,
+  "num_attention_heads": 16, # head_size = dim_att / num_attention_heads.
```

Comment:
Similar comment here. Calling these attention heads is highly misleading.

Reply:
I kind of disagree, as rwkv code generally references time mixing as `attention`, and the RWKV kernel is often called a type of "linear attention." But, I can add a bunch of configs to decouple rkwv and transformer config options, but this will just create a lot of config args that have essentially the same purpose in my opinion.